chromium_src/chrome/browser/themes/theme_service_factory.cc

```diff
@@ -20,11 +16,5 @@
 #endif
 
 #include "src/chrome/browser/themes/theme_service_factory.cc"
-#undef BRAVE_THEMESERVICEFACTORY_BUILDSERVICEINSTANCEFOR
 
-content::BrowserContext* ThemeServiceFactory::GetBrowserContextToUse(
-    content::BrowserContext* context) const {
-  if (context->IsTor())
-    return context;
```

Comment:
doesn't look like this override is needed now. Tor windows look correct.

this change was introduced here https://github.com/brave/brave-core/pull/6912

Reply:
Right, `BraveThemeService` doesn't have logic that check tor or not anymore.